staging/src/k8s.io/metrics/pkg/client/custom_metrics/fake/fake_client.go

```diff
@@ -52,13 +52,12 @@ func (i GetForActionImpl) GetSubresource() string {
 }
 
 func NewGetForAction(groupKind schema.GroupKind, namespace, name string, metricName string, labelSelector labels.Selector) GetForActionImpl {
-	mapping, err := api.Registry.RESTMapper().RESTMapping(groupKind)
-	if err != nil {
-		panic(fmt.Sprintf("unable to get REST mapping for groupKind %s while building GetFor action: %v", groupKind.String, err))
-	}
+	// the version doesn't matter
+	gvk := groupKind.WithVersion("")
+	gvr, _ := meta.UnsafeGuessKindToResource(gvk)
```

Comment:
@DirectXMan12 could you take a look at the last commit that fixed the fake client of custom_metrics? Thanks.